tutorials/cloud-run-with-pomerium-for-end-user-access/index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Cloud Run with Pomerium for end-user access
+description: This guide covers how to deploy Pomerium to Cloud Run and use it to protect other endpoints via Authorization Headers.
+author: desimone
+tags: Cloud Run, Pomerium
+date_published: 2020-08-13
+---
+
+# Cloud Run with Pomerium for end-user access
```

Comment:
I'd probably call this

# Authorizing end-users in Cloud Run with Pomerium

Reply:
👍